src/main/java/com/example/demo/services/impl/UserServiceImpl.java

```diff
@@ -51,20 +67,38 @@ public User getUserById(final long id) {
     }
 
     @Override
-    public void addUser(final UserCreation user) {
+    public boolean addUser(final UserCreation user) {
+        if (user.getAge() < 0) {
+            log.error(ageValidError);
+            throw new RuntimeException(ageValidError);
+        }
         userRepository.save(userEntityMapper.map(user));
+        return true;
```

Comment:
It does not make sense to return true, as otherwise you don't return fake you throw an exception